src/Generators/Microsoft.Gen.AutoClient/Emitter.cs

```diff
@@ -440,6 +462,12 @@ private void EmitStatics(RestApiType restApiType, string staticsName)
         OutLn($"private static class {staticsName}");
         OutOpenBrace();
 
+        OutLn("public static readonly char[] Slashes = { '/', '\\\\' };");
```

Comment:
Why is this public?

Reply:
It's part of `private static class {staticsName}` so if it's private the generated code won't be able to access it.

Reply:
This could be made internal, though the other properties from this private class are public.

Reply:
These have moved to the utilities class now.